Cubical/Data/Nat/Base.agda

```diff
@@ -44,3 +47,12 @@ isEven zero = true
 isEven (suc n) = isOdd n
 isOdd zero = false
 isOdd (suc n) = isEven n
+
+--Typed version
```

Comment:
Wouldn't it be nicer to define these by composing `isEven`/`isOdd` with a map sending `true` to `Unit` and `false` to `⊥`?

Reply:
Also the name is not very good... Maybe `isEvenType` or `isEvenT`?

Reply:
Sure, made isEven like that now. For isOdd, I do think it's nice to define it as isEven (suc n). That way, anything proved about isEven will automatically hold for isOdd.

Reply:
> PS: there are some white space issues in the summary file...

... as is tradition.